src/utils/validators/index.ts

```diff
@@ -23,3 +23,13 @@ export const validatePercentage = (percent: any) => {
 
 export const validateSlack = (v: string): boolean =>
   v.match("(#|@).+") !== null;
+
+/**
+ *  validatePatchId tests if a provided id is a mongo objectId indicating that it likely belongs to a patch and not a version
+ *
+ */
+//
+export const validatePatchId = (id: string): boolean => {
+  const mgobsonRegex = /^[a-f\d]{24}$/i;
```

Comment:
What rules is this using to determine if this is an ObjectId? Can we link them?

Reply:
This is the official regex the mongodb bson spec uses for objectids i've linked them in a comment